_docs/concepts/configuration/taxonomy.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Taxonomy
```

Comment:
I don't think this document needs to be on the website. Maybe in github, but maybe just keep as a google doc.

I'm worried it will be more confusing than helpful.

Reply:
Currently, we don't have a  functional definition of istio configures. It poses challenge when we communicate with our users, basically, ppl use the same term for different things. This tends to clearly define configuration terms for Istio users, also make sure references in other parts of the doc is consistent.